kuma/wiki/views/document.py

```diff
@@ -644,7 +647,10 @@ def document(request, document_slug, document_locale):
         if fallback_doc is not None:
             doc = fallback_doc
             if redirect_url is not None:
-                return redirect(redirect_url)
+                response = redirect(redirect_url)
+                if request.user.is_authenticated():
+                    add_never_cache_headers(response)
```

Comment:
My original intention was simply to be thorough in not caching when the user is logged-in, but you're right, I don't think this can vary for logged-in users. I will remove it.